src/fcntl.rs

```diff
@@ -205,6 +205,7 @@ libc_bitflags!(
     }
 );
 
+#[allow(missing_debug_implementations)]
```

Comment:
Why do you allow missing debug impls for certain types?

Reply:
No `libc` types implement `Debug`, so `derive(Debug)` doesn't work for them. I've decided to defer these implementations for now in order to get these checks working for PRs in general. I'd like to come back and add `impl Debug` for all types as a future PR.